### DISCUSSION TITLE
bug/ glitch scene/gui/text_edit.cpp:6172 - Index p_line_from = -1 is out of bounds (text.size() = 197).

### DISCUSSION THREAD

**Discussion by sourpotatobanana** [godotengine/godot-proposals#5507] [Editor]
when hover my mouse in red area , will get allot of same error, for some reason when enter more line its gone, but it will happen  again when i delete the line. how can i fix it? i try do to in new project but it wont happen.

![er](https://user-images.githubusercontent.com/42238031/192824048-7dd89bd0-dd03-458c-9dc7-a7d2ab7cf30c.PNG)

here the video that i try to explain.
https://youtu.be/STjFj6m6Jb4
update - i put wrong link before.

**Comment by Zireael07** [comment]
This is not the place to report bugs

**Reply by sourpotatobanana** [reply]
someone from discord tell me to open dicussions in github, i dont know where to put, mind show me? my first time using it

**Reply by Zireael07** [reply]
https://github.com/godotengine/godot/issues
